app/elements/submit-page.html

```diff
@@ -797,64 +817,47 @@ <h4 class="role__name"><img src="../images/icons/share.svg" alt="Share" class="r
-                    <li class="faq__requirement"><a href="http://unicefstories.org/" class="faq__link">Submit any questions you may have.</a> Responses will be posted regularly.<br>
-                        <em class="faq__emphasis">Please note that the period to submit questions for the REOI closes on 16 February, 2016.</em>
-                        <a href="" class="faq__button">Ask now</a>
-                        See the <a href="http://unicefstories.org/" class="faq__link">answers</a> to what others have asked <a href="http://unicefstories.org/" class="faq__link">&#8594;</a>
-                    </li>
+                    [[startups.submit_startup_step_content.2]]
                 </ul>
             </li>
             <li class="faq__step">
-                <h3 class="faq__name">Step 4</h3>
-                <strong class="faq__description">Submit</strong>
+                <h3 class="faq__name">[[startups.submit_startup_step_titles.3]]</h3>
+                <strong class="faq__description">[[startups.submit_startup_step_bold.3]]</strong>
                 <ul class="faq__list">
-                    <li class="faq__requirement faq__requirement--unstyled">
-                        <a href="http://unicefstories.org/" class="faq__link">Submit</a> your expression of interest by completing the form.<br>
-                        Preview the questions to prepare your submission, <a href="http://unicefstories.org/" class="faq__link">preview questions</a>
-                    </li>
+                    [[startups.submit_startup_step_content.3]]
                 </ul>
             </li>
         </ol>
         <div class="faq__submit">
-            <a href="https://docs.google.com/forms/d/e/1FAIpQLSdL7bKFc-zSzm4R4wK5eKsKRzW_HqQsdPYyA62sCXR84BrF9A/viewform?usp=send_form" class="faq__submit-link mdl-button mdl-js-button mdl-js-ripple-effect" target="_blank">Submission Form</a>
+            <a href="https://form.jotform.com/62704508221145" class="faq__submit-link mdl-button mdl-js-button mdl-js-ripple-effect" target="_blank">[[startups.submit_startup_submit_button]]</a>
         </div>
         <hr class="section__break--thin">
         <div class="summary">
             <img src="../images/icons/country_offices-alt.svg" alt="" class="summary__icon">
             <div class="summary__content">
-                <h3 class="summary__title">Got questions?</h3>
-                <p class="summary__description">Responses will be posted regularly.</p>
-                <em class="summary__description summary__emphassis">Please note that the period to submit questions for the REOI closes on DD MONTH, YYYY.</em>
-                <p class="summary__description">UNICEF will not answer any questions submitted after this date.</p>
-                <a href="http://unicefstories.org/" class="summary__link">Ask now</a>
+                <h3 class="summary__title">[[startups.submit_startup_conclusion_title]]</h3>
```

Comment:
Have you considered using app/elements/echo-html.html component? You can embed HTML with it.